pysumma/simulation.py

```diff
@@ -25,12 +25,11 @@ class Simulation(object):
     local_attributes: xr.Dataset = None
     parameter_trial: xr.Dataset = None
 
-    def __init__(self, executable, filemanager, run_suffix='_pysumma_run'):
+    def __init__(self, executable, filemanager):
```

Comment:
In here, do you think what variable do we have to define for `executable'? I think if we want to define executable itself such as executable="uwhydro/summa:sopron_2018", we need to delete every "docker_img" variable. If we don't delete it, "docker_img" defines inside some funtions. So there is no meaning to define executable in Simulation class.

Reply:
I well remove the `docker_img` variable throughout. I think it's better to just use `executable`.